### PR TITLE
Upgrade to gof3r 0.5.0

### PR DESCRIPTION
**Pull request by josegonzalez**
Also install missing ca-certificates so uploads complete successfully. Finally, I changed the timestamp to be UTC and also include the time.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Just added support for non-standard regions, so this should also close #4.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
-[![nodesource/node](http://dockeri.co/image/novacoast/s3backup)](https://registry.hub.docker.com/u/novacoast/s3backup/)
+[![nodesource/node](http://dockeri.co/image/dokkupaas/s3backup)](https://registry.hub.docker.com/u/dokkupaas/s3backup/)
 
 # Info
 Docker image that creates and streams a tar backup of a host volume to Amazon S3 storage.
 
 + Lightweight: Based on the [Alpine](https://github.com/gliderlabs/docker-alpine) base image
-+ Fast: Backups are streamed directly to S3 with [s3gof3r](https://github.com/rlmcpherson/s3gof3r)
++ Fast: Backups are streamed directly to S3 with [awscli](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html)
```
Comment (djdefi) [inline README.md]: I no longer maintain this project, however it seems like changing this to `awscli` defeats the purpose of this image, which is to utilize [`s3gof3r` to provide fast, parallelized, pipelined streaming access to Amazon S3](https://github.com/rlmcpherson/s3gof3r)

I have been out of the loop, but when this image was created, `awscli cp` couldn't do parallel requests, making it much slower for a large dataset. Do you happen to have any benchmarks comparing the two methods?

Comment (josegonzalez) [inline README.md]: I've given up on this PR. Initially I upgraded the image to use a later version of s3gof3r, but that project doesn't support regions other than us-east-1 well, so I switched to awscli.

According to [this post](https://aws.amazon.com/blogs/apn/getting-the-most-out-of-the-amazon-s3-cli/), it supports chunking large files and streaming them up internally.

**Comment by josegonzalez** [comment]
Closing this PR though since the original PR sort of sat here for a while and while I'd be happy to do the legwork of benchmarking, if it's not maintained, its not worth the effort :(

Thanks for commenting though!

**Comment by djdefi** [comment]
@josegonzalez I no longer personally maintain this, but someone from the Novacoast org could potentially pick it up. Didn't mean to discourage you from contributing!

**Comment by josegonzalez** [comment]
No worries, it doesn't seem like someone from the org is interested either. I managed to "fix" my use case by switching to awscli, which seems to work well enough.

**Pull request closed without merging**